website/content/api-docs/secret/gcp.mdx

```diff
@@ -182,7 +185,7 @@ $ curl \
 
 | Method | Path                        |
 | :----- | :-------------------------- | ------------------- |
-| `POST` | `/gcp/roleset/:name/rotate` | `204 (empty body)`` |
+| `POST` | `/gcp/roleset/:name/rotate` | `204 (empty body)`  |
```

Comment:
I can move this to a separate PR if you'd rather. They are typos and copy-paste errors.